oak-api/src/main/java/org/apache/jackrabbit/oak/api/StrictPathRestriction.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.jackrabbit.oak.api;
+
+public enum StrictPathRestriction {
+    ENABLE("enable"),
+    WARN("warn"),
+    DISABLE("disable");
+    private String stringValue;
```

Comment:
Do you use the stringValue somewhere? I couldn't find... If not, could you remove it.